### ISSUE TITLE
Display chat messages from a moderator using a different colour

### ISSUE THREAD

**Issue by ffdixon** [bigbluebutton/bigbluebutton#4858]
To make it easier to see the Instructor's (moderators) messages in the chat, assign moderator's messages a different colour.

**Comment by tylercopeland** [comment]
@ffdixon instead of introducing a third colour in the chat module, I was thinking we could leverage existing colours and text to make the moderator's messages stand out. 

The in the example below, you'll see that I've used blue for the moderator's name, added the word "moderator" to the right and added a light blue background.  

<img width="338" alt="screen shot 2017-12-22 at 3 15 06 pm" src="https://user-images.githubusercontent.com/1273169/34311355-2bc1cce0-e72b-11e7-81da-02e34adb2ffb.png">

**Comment by ffdixon** [comment]
We could try something similar -- making the user's name bold if they are a moderator (same colour as they are now).

**Tyler**
Welcome to the class

Student
Glad to be here

**Comment by oswaldoacauan** [comment]
Maybe add a background color?

**Comment by tylercopeland** [comment]
@ffdixon I explored that simple approach during design, but I left like it wasn't "enough" to suggest that "Tyler" was any different than "Student". That feeling is what drove me to explore other options, such as including "moderator" to the right of the name and the use of colour. 

If my approach has too complex, an iteration that might be a happy medium is to include the moderator icon to the right of the moderators name. 

**Tyler** [icon]
Welcome to the class

Student
Glad to be here

Thoughts?

**Comment by ffdixon** [comment]
I think we should try the icon -- here's what it looks like when bold.

![image](https://user-images.githubusercontent.com/290351/35006540-daa3ce5a-fac4-11e7-93db-f7afb2b9f6ef.png)

**Comment by tylercopeland** [comment]
@ffdixon here's an updated design that includes an icon, as well as some a small adjustment to the colours used for the participant's name and message (these new colours reflect what's being used in the HTML5 client). 

![screen shot 2018-01-16 at 2 37 40 pm](https://user-images.githubusercontent.com/1273169/35008536-d73dc242-faca-11e7-8ff2-822894c245ae.png)

**Comment by capilkey** [comment]
Bold means moderator and the icon means presenter?

**Comment by tylercopeland** [comment]
@capilkey I used the wrong icon in the previous design. If you have a look above, I've updated the screenshot to use the moderator icon.

Something to think about is if we're updating/tweaking the look and feel for how the moderator looks in the chat, we may want to do the same for presenters. 

There following is my recommendations:

**Moderator Name [Moderator Icon]**
Text

**Presenter Name [Presenter Icon]**
Text

Student
Text
